caasp-multi-master-stack.yaml

```diff
@@ -104,6 +117,48 @@ parameters:
     description: Root Password for the VMs
     default: linux
 
+  cpi_auth_url:
+    type: string
+    description: Cloud Provider Interface - Keystone API URL
+    default: https://api.keystone.example.net:5000/v3
```

Comment:
Why are we setting the default to something that is always going to be wrong?

Reply:
The user still has chance to change it to the appropriate one, during the  CaaSP admin  node bootstrapping step (its available as a text field in the UI, there is a hint as to where the user can go and get the appropriate endpoint, and is mentioned in the docs as well). Unfortunately heat does not currently give access to this pubilc endpoint, unless the endpoint is created by the heat template, so we cannot set it automagically.